Library/Formula/kapacitor.rb

```diff
@@ -0,0 +1,193 @@
+require "language/go"
+
+class Kapacitor < Formula
+  desc "Open source time series data processor"
+  homepage "https://influxdb.com"
+  url "http://github.com/influxdb/kapacitor/archive/v0.1.1.tar.gz"
+  sha256 "4e67f98fec04951490b0820d18d7579413d6495d8629d47fa233697688c03236"
+
+  head do
+    url "https://github.com/influxdb/kapacitor.git"
+  end
```

Comment:
You can do that in one line: `head "https://github.com/influxdb/kapacitor.git"`.